thirdeye/thirdeye-frontend/app/pods/components/events-table/component.js

```diff
@@ -52,57 +52,62 @@ export default Component.extend({
   ),
 
   // Require to display a loader for long rendering
-  didUpdateAttrs(...args) {
+  didUpdateAttrs() {
     later(() => {
-      this._super(args);
+      this._super(...arguments);
     });
   },
 
-  /**
-   * Columns required by ember-models-table
-   */
-  columns: [
-    {
-      template: 'custom/checkbox',
-      useFilter: false,
-      mayBeHidden: false,
-      className: 'events-table__column--checkbox'
-    },
-    {
-      propertyName: 'displayLabel',
-      template: 'custom/eventLabel',
-      title: 'Event Name',
-      className: 'events-table__column'
-    },
-    {
-      propertyName: 'eventType',
-      title: 'Type',
-      filterWithSelect: true,
-      sortFilterOptions: true,
-      className: 'events-table__column--compact'
-    },
-    {
-      propertyName: 'humanRelStart',
-      title: 'Start',
-      className: 'events-table__column--compact',
-      sortedBy: 'relStart',
-      disableFiltering: true
-    },
-    {
-      propertyName: 'humanDuration',
-      title: 'Duration',
-      className: 'events-table__column--compact',
-      sortedBy: 'duration',
-      disableFiltering: true
-    }
+  init() {
+    this._super(...arguments);
+
+    /**
+     * Columns required by ember-models-table
+     */
+    const columns = [
```

Comment:
This init is growing too large. Can we move this outside of init? Like at the top? Or better yet, some const file?